src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/serializer.html.twig

```diff
@@ -0,0 +1,228 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block menu %}
+    <span class="label {{ not collector.handledCount ? 'disabled' }}">
+        <span class="icon">{{ include('@WebProfiler/Icon/validator.svg') }}</span>
```

Comment:
Icon needs to be updated. We are working on updating all icons soon, so let's not block this PR for now.